model.go

```diff
@@ -13,9 +13,9 @@ const schema = "currencies"
 type Quote struct {
 	//lint:ignore U1000 hit for go-pg
 	tableName     struct{} `pg:"currencies.quotes"`
-	Height        int64    `pg:",pk,notnull"`
-	Price         int64    `pg:",notnull"`
-	VolumeBase24h int64    `pg:"volume_base_24h,notnull"`
+	Height        int64    `pg:",pk,notnull,use_zero"`
```

Comment:
I don't think we should ever insert a 0 height

Reply:
I'm happy to remove this from the height, but if it were 0 for {whatever} reason, it would fail in the same way. The alternative approach would be to fix the default values for these columns in the schema, but I don't see where that's managed. I think we should do one or the other. Let me know how you'd like the schema change applied if we're going the other way here.